ProcessLib/LIE/HydroMechanics/HydroMechanicsProcess.cpp

```diff
@@ -345,6 +345,22 @@ void HydroMechanicsProcess<GlobalDim>::initializeConcreteProcess(
         mesh_prop_fracture_shear_failure->resize(mesh.getNumberOfElements());
         _process_data.mesh_prop_fracture_shear_failure =
             mesh_prop_fracture_shear_failure;
+
+        auto mesh_prop_nodal_w =
+            const_cast<MeshLib::Mesh&>(mesh)
+                .getProperties()
+                .template createNewPropertyVector<double>(
+                    "nodal_w", MeshLib::MeshItemType::Node, GlobalDim);
+        mesh_prop_nodal_w->resize(mesh.getNumberOfNodes() * GlobalDim);
```

Comment:
Better to use `MeshLib::getOrCreateMeshProperty<double>(mesh, "nodal_w", MeshLib::MeshItemType::Node, GlobalDim)` which would return an existing mesh property in case it already exists. `createNewPropertyVector` would return a nullptr  in that case.